docs/_config.yml

```diff
@@ -4,4 +4,13 @@ defaults:
       path: "" # an empty string here means all files in the project
     values:
       layout: "default"
+      title: Expensify Help
+      tagline: Find all the answers to all your Expensify questions, as well as support for user and admin questions from the Expensify Community
+      description: The Community is a great place for Expensify users and experts to help one another, access training and implementation advice, discuss best practice approaches and contribute their feature and functionality ideas.
+      url: help.expensify.com
+      author: Expensify
+      logo: /assets/images/expensify-help.svg
 exclude: [README.md, TEMPLATE.md]
+
+plugins:
+  - jekyll-seo-tag
```

Comment:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/47436092/195921582-f8b52956-3266-471b-8984-b2262078a79e.png">

:eye_twitch:

Reply:
Add a newline at the end of the file plz